app/code/Magento/SendFriend/Block/Plugin/Catalog/Product/View.php

```diff
@@ -6,29 +6,32 @@
 
 namespace Magento\SendFriend\Block\Plugin\Catalog\Product;
 
+use Magento\Catalog\Block\Product\View as ProductView;
+use Magento\SendFriend\Model\SendFriend;
+
 class View
 {
     /**
-     * @var \Magento\SendFriend\Model\SendFriend
+     * @var SendFriend
      */
     protected $_sendfriend;
 
     /**
-     * @param \Magento\SendFriend\Model\SendFriend $sendfriend
+     * @param SendFriend $sendfriend
      */
     public function __construct(
-        \Magento\SendFriend\Model\SendFriend $sendfriend
+        SendFriend $sendfriend
```

Comment:
Why not promote this property?
```suggestion
        protected readonly SendFriend $_sendfriend,
```